lib/content_security_policy/middleware.rb

```diff
@@ -12,12 +12,12 @@ def call(env)
       _, headers, _ = response = @app.call(env)
 
       return response unless html_response?(headers)
-      ContentSecurityPolicy.base_url = request.host_with_port if !Rails.env.production?
+      base_url = request.host_with_port
```

Comment:
Is there any ways `request.host_with_port` could be "manipulated" by the users or is always under "our" control?